packages/create-cloudflare/src/__tests__/common.test.ts

```diff
@@ -42,23 +42,25 @@ describe("isGitConfigured", () => {
 });
 
 describe("validateProjectDirectory", () => {
+	const args = {};
```

Comment:
could you expand these tests to also test that if the existingScript flag is provided otherwise invalid names are allowed?